vulnfeeds/cmd/combine-to-osv/main.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/google/osv/vulnfeeds/cves"
+	"github.com/google/osv/vulnfeeds/vulns"
+)
+
+const (
+	defaultCvePath        = "cve_jsons"
+	defaultPartsInputPath = "parts"
+	defaultOsvOutputPath  = "osv_output"
+)
+
+func main() {
+	cvePath := flag.String("cvePath", defaultCvePath, "Path to CVE file")
+	partsInputPath := flag.String("partsPath", defaultPartsInputPath, "Path to CVE file")
+	osvOutputPath := flag.String("osvOutputPath", defaultOsvOutputPath, "Path to CVE file")
+	flag.Parse()
+
+	err := os.MkdirAll(*cvePath, 0755)
+	if err != nil {
+		log.Fatalf("Can't create output path: %s", err)
+	}
+	err = os.MkdirAll(*osvOutputPath, 0755)
+	if err != nil {
+		log.Fatalf("Can't create output path: %s", err)
+	}
+
+	allCves := loadAllCVEs(*cvePath)
+	allParts := loadParts(*partsInputPath)
+	combinedData := combineIntoOSV(allCves, allParts)
+	writeOsvFile(combinedData, *osvOutputPath)
+}
+
+func loadParts(partsInputPath string) map[string][]vulns.PackageInfo {
```

Comment:
Could you please add a descriptive comment per function?

See also https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences